test-projects/integration/charts.visivo.yml

```diff
@@ -0,0 +1,44 @@
+selectors:
+  - name: Common Selector
+    default: Simple Line
+    is_multi: false
```

Comment:
Nice, this is good to be able to split it out. Then we can include it in dashboards as an independent item.